packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -729,6 +731,855 @@ function ResizableTable() {
+
+      /*- begin highlight -*/
+      // Create random id to allow for multiple copies of the same item
+      let processedItems = await Promise.all(
+        items.map(async (item) => (
+          {...JSON.parse(await (item as TextDropItem).getText('custom-app-type-copy-default')), id: Math.random().toString(36).slice(2)}
+        ))
+      );
+      /*- end highlight -*/
+      list.append(...processedItems);
+    }
+  });
+
+
+  return (
+    <TableView
+      aria-label="Droppable TableView in default copy operation example"
+      width="size-5000"
+      height="size-2400"
+      dragAndDropHooks={dragAndDropHooks}>
+      <TableHeader columns={columns}>
+        {column => (
+          <Column
+            key={column.id}
+            align={column.id === 'date' ? 'end' : 'start'}>
+            {column.name}
+          </Column>
+        )}
+      </TableHeader>
+      <TableBody items={list.items}>
+        {item => (
+          <Row>
+            {columnKey => <Cell>{item[columnKey]}</Cell>}
+          </Row>
+        )}
+      </TableBody>
+    </TableView>
+  );
+}
+
+function DragIntoTablesDefaultCopy() {
+  let columns = [
+    {name: 'Name', id: 'name'},
+    {name: 'Type', id: 'type'},
+    {name: 'Date Modified', id: 'date'}
+  ];
+
+  let sourceList = useListData({
+    initialItems: [
+      {id: '1', type: 'file', name: 'Adobe Photoshop', date: '6/7/2020'},
+      {id: '2', type: 'file', name: 'Adobe XD', date: '4/7/2021'},
+      {id: '3', type: 'file', name: 'Adobe InDesign', date: '11/20/2010'},
+      {id: '4', type: 'file', name: 'Adobe AfterEffects', date: '1/18/2016'}
+    ]
+  });
+
+  let targetList = useListData({
+    initialItems: [
+      {id: '5', type: 'file', name: 'Adobe Dreamweaver', date: '6/7/2020'},
+      {id: '6', type: 'file', name: 'Adobe Fresco', date: '2/17/2020'},
+      {id: '7', type: 'file', name: 'Adobe Connect', date: '1/23/2020'},
+      {id: '8', type: 'file', name: 'Adobe Lightroom', date: '12/17/2020'}
+    ]
+  });
+
+
+  return (
+    <Flex wrap gap="size-300">
+      <DraggableTableViewDefaultCopy list={sourceList} columns={columns} />
+      <DroppableTableViewDefaultCopy list={targetList} columns={columns} />
+    </Flex>
+  );
+}
+```
+</details>
+
+```tsx example
+<DragIntoTablesDefaultCopy />
+```
+
+### Custom drag previews
+
+By default, the drag preview for a dragged row will include the `textValue` provided to that row.
+If no `textValue` was provided, it will default to the contents of the columns that are row headers, as specified by the `isRowHeader` prop on columns.
+If no row header columns were specified, the first column will be used, as mentioned in the [Accessibility](#accessibility) section.
+
+Use the `renderPreview` prop to provide a custom drag preview. `keys` and `draggedKey` are passed to this function, where `keys` includes all the keys of the items being dragged, and `draggedKey` is the key of the item the user initiated the drag from.
+
+```tsx example
```

Comment:
Should this be hidden with a "show code" too?
```suggestion
```tsx example export=true render=false
```